README.md

```diff
@@ -62,24 +62,26 @@ launch them through:
 
 #### Running Tests in docker
 
-Once your docker-compose environment is running, you can run the test runner image:
-
-    $ docker-compose run --rm testrunner
-
-Now you are in a bash shell. You can now run tests as you would do in your local environment:
-
-    $ tox -e '{py35,py36}-redis{210}'
-
-We also provide a shell script to execute commands in the provided container.
-
-For example to run the tests for `redis-py` 2.10 on Python 3.5 and 3.6:
-
-    $ ./scripts/ddtest tox -e '{py35,py36}-redis{210}'
-
-If you want to run a list of tox environment (as CircleCI does) based on a
-pattern, you can use the following command:
-
-    $ scripts/ddtest scripts/run-tox-scenario '^futures_contrib-'
+Once your docker-compose environment is running, you can use the shell script to
+execute tests within a Docker image. You can start the container with a bash shell: 
+
+    $ scripts/ddtest
+
+You can now run tests as you would do in your local environment. We use
```

Comment:
Might be worth mentioning that we don't install tox or riot in the base image (iirc)

Reply:
We do with `scripts/ddtest` so it works out of the box.